### PR TITLE
Create default project nodeSelector for NPD to run on all nodes (including masters)

### PR DESCRIPTION
**Pull request by joelsmith**
See https://bugzilla.redhat.com/show_bug.cgi?id=1580286

### CODE REVIEW

File: roles/openshift_node_problem_detector/tasks/install.yaml [hunk 1]
```diff
@@ -8,6 +8,8 @@
   oc_project:
     name: openshift-node-problem-detector
     state: present
+    node_selector:
+      - ""
```
Comment (vrutkovs) [inline roles/openshift_node_problem_detector/tasks/install.yaml]: The line should have 4 spaces to pass yaml check

**Comment by joelsmith** [comment]
@vrutkovs Could you please re-review? Or @ingvagabund  ?

**Pull request merged**